### ISSUE TITLE
ReferenceError: value is not defined

### ISSUE THREAD

**Issue by whilejoe** [doczjs/docz#1139]
## Bug Report

It looks like a regression was introduced in https://github.com/doczjs/docz/commit/4edf18becab4f9e91b0c1c2773a99052f333dddd#diff-f2ab6a7df426a3569049338f793a31caR14

**`value` is referenced, but not defined:**

```js
const getDefaultValue = ({ defaultValue, type }) => {
  if (!defaultValue.value || !type.name) return null
  if (defaultValue.value === "''") {
    return <em>[Empty string]</em>
  }
  if (type.name === 'string') {
    return <em>{value.replace(/\'/g, '"')}</em>
  }
  if (typeof defaultValue.value === 'object' && defaultValue.value.toString) {
    return defaultValue.value.toString()
  }
  return <em>{defaultValue.value}</em>
}
```

**To Reproduce**

1. Have a `defaultProp` value of type `string`

```jsx
TextInput.defaultProps = {
  type: 'text'
};
```
2. Use `Props` components in an `mdx` file
```mdx
import { Props } from 'docz';

<Props of={TextInput}></Props>
```

3. Run npm run docz dev
4. See error in browser

```shell
Uncaught ReferenceError: value is not defined
    at getDefaultValue (index.js:15)
    at Prop (index.js:39)
    at Prop (react-hot-loader.development.js:827)
    at renderWithHooks (react-dom.development.js:16320)
    at mountIndeterminateComponent (react-dom.development.js:18735)
    at beginWork$1 (react-dom.development.js:20084)
    at HTMLUnknownElement.callCallback (react-dom.development.js:362)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:411)
    at invokeGuardedCallback (react-dom.development.js:466)
    at beginWork$$1 (react-dom.development.js:25730)
    at performUnitOfWork (react-dom.development.js:24638)
    at workLoopSync (react-dom.development.js:24614)
    at performSyncWorkOnRoot (react-dom.development.js:24182)
    at react-dom.development.js:12238
    at unstable_runWithPriority (scheduler.development.js:643)
    at runWithPriority$2 (react-dom.development.js:12188)
    at flushSyncCallbackQueueImpl (react-dom.development.js:12233)
    at flushSyncCallbackQueue (react-dom.development.js:12221)
    at scheduleUpdateOnFiber (react-dom.development.js:23601)
    at Object.enqueueSetState (react-dom.development.js:14029)
    at LocationProvider.../node_modules/react/cjs/react.development.js.Component.setState (react.development.js:335)
    at index.js:104
```

**Expected behavior**

Default prop value can be a string without error.

**Environment**

- docz version: [^2.0.0-rc.44]
- OS: [OSX 10.14.6]
- Node/npm version: [10.16.3, 6.11.3]

**Comment by rakannimer** [comment]
Hey,

Thanks for catching and reporting this issue, 

It should be fixed in `2.0.0-rc.48`. Can you give it a try ?

**Comment by whilejoe** [comment]
Looking good now. Thanks for the quick fix @rakannimer!
